oidc-controller/api/core/config.py

```diff
@@ -69,6 +67,8 @@ class GlobalConfig(BaseSettings):
     JWT_ALGORITHM = "HS256"
     JWT_ACCESS_TOKEN_EXPIRE_MINUTES = 300
 
+    CONTROLLER_API_KEY: str = os.environ.get("CONTROLLER_API_KEY", "change-me")
```

Comment:
Should we default to no api key if none is provided (e.g.: unprotected)?
It might be an edge case, but I think it makes sense.